interfaces/sourcegen/src/sourcegen/csharp/generator.py

```diff
@@ -83,7 +81,7 @@ def _get_property_text(self, clib_area: str, c_name: str, cs_name: str,
             clib_area + "3_set" + c_name.capitalize(),
             CsFunc("", "", "", "", ""))
 
-        if prop_type in ["int", "double"]:
+        if prop_type in ["int32_t", "double"]:
```

Comment:
Here we want to keep the original line, as at this point source gen is talking in C# type names, and `int` is correct.
```suggestion
        if prop_type in ["int", "double"]:
```

One completely optional thing we could do if it would help clarity is the the .NET type names which are always different from C type names and might make it clear in context. (The C# type names are aliases for the .NET official type names, which consistently use PascalCase amongst other things)
| C# | .NET |
| - | - |
| `int` | `System.Int32` |
| `long` | `System.Int64` |
| `double` | `System.Double` |
| `string` | `System.String` |

(`System` namespace prefix is usually implied)

Reply:
To keep things simple, I'd suggest to retain the C# names.